components/automate-cli/cmd/chef-automate/applications.go

```diff
@@ -46,6 +73,65 @@ func newApplicationsDisableCmd() *cobra.Command {
 	}
 }
 
+func runApplicationsListDisconnectedServicesCmd(*cobra.Command, []string) error {
+	if thresholdMinutes <= 0 {
+		return status.Errorf(status.InvalidCommandArgsError,
+			"%d is not a valid threshold time in minutes. The expected time must be greater than zero.",
+			thresholdMinutes,
+		)
+	}
+
+	if format != "json" && format != "pretty" {
+		return status.Errorf(status.InvalidCommandArgsError,
+			"%s is not a valid format type. Available formats are 'json' and 'pretty'.",
+			format,
+		)
+	}
+
+	var (
+		ctx            = context.Background()
+		apiClient, err = apiclient.OpenConnection(ctx)
+	)
+	if err != nil {
+		return status.Wrap(err, status.APIUnreachableError,
+			"Failed to create a connection to the API")
+	}
+
+	servicesRes, err := apiClient.ApplicationsClient().GetDisconnectedServices(ctx,
+		&applications.DisconnectedServicesReq{
+			ThresholdMinutes: int32(thresholdMinutes),
+		},
+	)
+	if err != nil {
+		return err
+	}
+
+	if len(servicesRes.Services) == 0 {
+		writer.Printf(
+			"There are no disconnected services with a threshold of %d minute(s)\n",
+			thresholdMinutes,
+		)
+		return nil
+	}
+
+	switch format {
+	case "json":
+		json, err := (&jsonpb.Marshaler{OrigName: true}).MarshalToString(servicesRes)
+		if err != nil {
+			return err
+		}
+		writer.Println(json)
```

Comment:
Haven't checked, but doesn't this return an error, too?

Reply:
No that I have seen.